### PR TITLE
Launcher folder scanning

### PR DESCRIPTION
**Pull request by TheMulti0**

### CODE REVIEW

File: GameScanner/Folder/LauncherFolderScanner.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        private CancellationToken _token;
+
+        public event EventHandler<GameFolderEventArgs> GameFolderFound;
+
+        public LauncherFolderScanner(string launcherFolders)
+        {
+            _launcherFolders = PathToGameFolders(File.ReadAllLines(launcherFolders));
+        }
+
+        public LauncherFolderScanner(IEnumerable<string> launcherFolders)
```
Comment (DynamicField) [inline GameScanner/Folder/LauncherFolderScanner.cs:24]: What about `params string[] launcherFolders`?

File: GameScanner/Folder/LauncherFolderScanner.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            CheckDirectoryChildren(directory);
+            return false;
+        }
+
+        private void CheckDirectoryChildren(DirectoryInfo directory)
+        {
+            try
+            {
+                if (directory.EnumerateDirectories()
+                   .Any())
```
Comment (DynamicField) [inline GameScanner/Folder/LauncherFolderScanner.cs:115]: You can put it on the same line as `EnumerateDirectories` :)

**Review by DynamicField — APPROVED** [review]
Noicy 👌

File: GameScanner/Folder/LauncherFolderScanner.cs (hunk 2 above)
Comment (TheMulti0) [reply GameScanner/Folder/LauncherFolderScanner.cs:115]: true. Resharper is the one to blame there :)

**Pull request merged**